fiasco/util/util.py

```diff
@@ -88,7 +88,7 @@ def get_chianti_catalog(ascii_dbase_root):
     -------
     : `dict`
         All CHIANTI files, separated by category. The resulting dictionary should have the
-        following keys: 'abundance_files', 'ioneq_files', 'ip_files', 'continuum_files',
+        following keys: 'abundance_files', 'ionization_files', 'ip_files', 'continuum_files',
```

Comment:
```suggestion
```

I think this just wasn't updated after going back to keeping "ioneq" lower down in this function